playbooks/roles/tower/tasks/bootstrap_credentials.yml

```diff
@@ -36,6 +36,8 @@
     description: '{{ tower_credential_bundle_default_desc }}'
     organization: '{{ tower_organization }}'
     state: present
+    become_method: '{{ tower_credential_bundle_become_method }}'
```

Comment:
@alanmoran I'm not sure if this will work? The SRE dev tower is an ec2 instance but Eng and QE towers are running containers on Openshift. Sudo won't be available from a container

Reply:
It shouldn't affect ENG/QE Towers though unless they have a task with `become:true`. Only then does the become_method get used.

Reply:
Ok cool, we can test it in any case

Reply:
@pmccarthy Is this something you can test quickly on your side? I put this PR separate as its the only blocker to us bootstrapping SRE prod tower. The other PR can be deferred if we absolutely need to as it is only adding the existing jobs to this repo.

Reply:
@pmccarthy Did you get a chance to test this after?

Reply:
@alanmoran I didn't, but can look to do this at some point today for you